Pid.hpp

```diff
@@ -15,11 +17,20 @@ class Pid {
     /** clear any windup */
     void clearWindup();
 
+
+    //tuner stuff
+    void initialize_tuner();
+
+    void set_from_tuner();
+
+    void start_cycle();
+    void run();
```

Comment:
Could you change this function name to like runWithTuner so it doesn't get confused with the origional run function.